src/index.ts

```diff
@@ -11,7 +11,9 @@ import { getApp } from './app';
 let depContainer: DependencyContainer | undefined;
 
 void getApp()
-  .then(([app, depContainer]) => {
+  .then(([app, container]) => {
```

Comment:
run prettier formatting, also consider adding a test case where you mock some error in `getApp` and expect ON_SIGNAL to be called.